### PR TITLE
Changed admins and updated mangainers. Also scrubbed teams

### LINKED ISSUE #2: Define UOR initial organization configuration
[Peribolos](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/peribolos) is currently configured to manage the configuration of the GitHub organization. As part of the initialization, the existing organization configuration was captured into a [peribolos.yaml](https://github.com/uor-framework/.github/blob/main/peribolos.yaml) file.

To support the longevity of the project and to scale effectively, the following configurations should be applied:

1. The following users should be administrators of the organization. Other users who are currently configured as administrators should be moved to a member role
    1. @sabre1041 
    2. @afflom 
    3. @jpower432 
2. To support a growing Open Source community, teams should be created to align with the initial key areas of the project. All other existing teams should be removed. No members are to be defined at this time as this task is the skeleton to support future efforts
    1. `client`
        1. `client-maintainers`
    2. `event`
        1. `event-maintainers`
    3. `ci`
        1. `ci-maintainers`
    4. `docs`
        1. `docs-maintainers`

### PR DESCRIPTION
**Pull request by christianh814**
Fixes #2 

Signed-off-by: Christian Hernandez <christian@codefresh.io>

### CODE REVIEW

File: peribolos.yaml [hunk 1]
```diff
@@ -52,9 +52,15 @@ orgs:
         description: Public GH Pages Repository
         has_projects: true
     teams:
-      maintainer:
-        description: maintainers team
-        maintainers:
-          - usrbinkat
-          - jpower432
+      ci:
```
Comment (sabre1041) [inline peribolos.yaml:55]: Can the subteams for maintainers also be included?

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Thanks @christianh814 for kicking this off.

As noted in the specific in line comment, would you be able to also include the sub teams for maintainers? The permissions associated with the maintainers subteam will be different than the general team

File: peribolos.yaml (hunk 1 above)
Comment (christianh814) [reply peribolos.yaml:55]: > Can the subteams for maintainers also be included?

Latest update should have fixed this. PTAL

https://github.com/uor-framework/.github/pull/3/commits/c409e6fb70ee72eb8d8b4762c3635432cbe23275

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**